basler_params:
  -
    name: AcquisitionFrameRate
    type: float
    value: 1.0
  -
    name: AcquisitionMode
    type: enum
    value: Continuous
  -
    name: ExposureAuto
    type: enum
    value: Continuous
